lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -9,6 +9,10 @@
 
 $nx-filter-input-height: 20px;
 
+.nx-filter-input, .nx-filter-input * {
```

Comment:
This could also be added to the style below via:
`&, & * {`
this has the benefit of keeping the styles grouped.

Reply:
```
.nx-filter-input {
  @include container-horizontal;

  display: inline-block;
  vertical-align: middle;
  white-space: nowrap;

  &, & * {
    box-sizing: content-box;
  }
}
```